packages/svelte/src/internal/client/runtime.js

```diff
@@ -460,7 +465,24 @@ export function update_effect(effect) {
 function infinite_loop_guard() {
 	if (flush_count > 1000) {
 		flush_count = 0;
-		e.effect_update_depth_exceeded();
+		if (DEV) {
+			try {
+				e.effect_update_depth_exceeded();
+			} catch (e) {
+				// stack is garbage, ignore. Instead add a console.error message.
+				// eslint-disable-next-line no-console
+				console.error(
+					'Last ten effects were: ',
+					dev_effect_stack.slice(-10).map((d) => d.fn)
+				);
+				dev_effect_stack = [];
+				// @ts-expect-error
+				e.stack = '';
```

Comment:
I believe this needs to be `defineProperty` see `handle_error` in `runtime.js`

Reply:
I saw that you did that in other places, but I didn't understand why it is necessary since it doesn't error for me - could you explain?

Reply:
`stack` wasn't writable in some contexts and triggered an error of its own